api/Cinematica.API/src/Cinematica.API/Controllers/AuthController.cs

```diff
@@ -24,14 +28,15 @@ public class AuthController : ControllerBase
             RegionEndpoint.GetBySystemName(APP_CONFIG.GetValue<string>("Region"))
```

Comment:
You can access string values a bit easier from IConfiguration objects:
`RegionEndpoint.GetBySystemName(APP_CONFIG["Region"])`